x-pack/plugins/synthetics/public/apps/synthetics/components/common/screenshot/empty_image.tsx

```diff
@@ -32,16 +36,21 @@ export const imageStyle = css`
 `;
 
 export const EmptyImage = ({
+  size,
   isLoading = false,
-  width = IMAGE_WIDTH,
-  height = IMAGE_HEIGHT,
+  width,
+  height,
 }: {
   isLoading: boolean;
   width?: number;
   height?: number;
+  size?: EuiImageProps['size'];
```

Comment:
[nit] Since we are only using the `m` value I think we should specify only that in the type.